source/localizable/routing/rendering-a-template.md

```diff
@@ -20,15 +19,18 @@ template. For example, the `posts.new` route will render its template into the
 `posts.hbs`'s `{{outlet}}`, and the `posts` route will render its template into
 the `application.hbs`'s `{{outlet}}`.
 
-If you want to render a template other than the default one, implement the
-[`renderTemplate()`][1] hook:
-
-[1]: http://emberjs.com/api/classes/Ember.Route.html#method_renderTemplate
+If you want to render a template other than the default one, set the route's [`templateName`][1] property to the name of
+the template you want to render instead.
 
 ```app/routes/posts.js
 export default Ember.Route.extend({
-  renderTemplate() {
-    this.render('favoritePosts');
-  }
+  templateName: 'posts/favorite-posts'
 });
 ```
+
+You can override the [`renderTemplate()`][2] hook if you want finer control over template rendering.
+Among other things, it allows you to choose the controller used to configure the template and specific outlet to render
+it into.
```

Comment:
@locks @toddjordan mentioned that it helps translation if every period was followed by a newline. I assume you're talking about the newline following line 31.